docs/CLI/index.md

```diff
@@ -17,30 +17,30 @@ Install from the latest [release artifacts](https://github.com/oras-project/oras
 ### Linux
 
 ```bash
-curl -LO https://github.com/oras-project/oras/releases/download/v0.16.0/oras_0.16.0_linux_amd64.tar.gz
+curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_linux_amd64.tar.gz
 mkdir -p oras-install/
-tar -zxf oras_0.16.0_*.tar.gz -C oras-install/
+tar -zxf oras_1.0.0-rc.1_*.tar.gz -C oras-install/
 mv oras-install/oras /usr/local/bin/
-rm -rf oras_0.16.0_*.tar.gz oras-install/
+rm -rf oras_1.0.0-rc.1_*.tar.gz oras-install/
 ```
 
 ### macOS
 
 ```bash
-curl -LO https://github.com/oras-project/oras/releases/download/v0.16.0/oras_0.16.0_darwin_amd64.tar.gz
+curl -LO https://github.com/oras-project/oras/releases/download/v1.0.0-rc.1/oras_1.0.0-rc.1_darwin_amd64.tar.gz
 mkdir -p oras-install/
-tar -zxf oras_0.16.0_*.tar.gz -C oras-install/
+tar -zxf oras_1.0.0-rc.1_*.tar.gz -C oras-install/
 mv oras-install/oras /usr/local/bin/
```

Comment:
Yes, mac also needs `sudo`.
```suggestion
sudo mv oras-install/oras /usr/local/bin/
```
According to [System Integrity Protection](https://support.apple.com/en-us/HT204899), we are still able to write to `/usr/local/`